D/DuckDB/bundled/patches/0001-right-case-for-winsock2.h.patch

```diff
@@ -0,0 +1,26 @@
+From eab00b8a2cbb8c1917c05175de135d8411f6444f Mon Sep 17 00:00:00 2001
+From: Kimmo Linna <kimmo.linna@gmail.com>
```

Comment:
We're lucky someone already submitted this patch upstream :slightly_smiling_face:

Reply:
Someone should add those two other patches two. ;)